dart/biomechanics/DynamicsFitter.cpp

```diff
@@ -10628,6 +10629,10 @@ void DynamicsFitter::zeroLinearResidualsOnCOMTrajectory(
             = regularizeUnobservedTimesteps;
       }
 
+      int scaleGRFRow
+          = (numTimesteps * 3) + (numForcePlates * 3) + (numMissingSteps * 3);
+      trialLinearMapToPositions(scaleGRFRow, scaleGRFCol) = regularizeScaleGRF;
```

Comment:
Add the regularization value to the matrix, which just directly multiplies the `scaleGRF` on the input and tries to make it match 0 as part of the least squares fit.